pkgs/applications/version-management/radicle-cli/default.nix

```
@@ -18,28 +18,15 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "radicle-cli";
-  version = "0.6.1";
+  version = "1.0.0-rc8";
```

```suggestion
  version = "1.0.0-rc.8";
```

For consistency with upstream.